src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -11,7 +12,11 @@
      * Вернуть сумму нечетных числел или 0, если таких несуществует</p>
      **/
     public Integer oddSum(List<Integer> numbers) {
-        return null;
+        return numbers
+                .stream()
+                .mapToInt(x -> x)
```

Comment:
do we need to map int to int?

Reply:
Yes it needed to use sum().

Reply:
try to use .reduce(0, Integer::sum); then